src/database/backends/mongo.ts

```diff
@@ -0,0 +1,85 @@
+import * as env from "env-var";
+import {
+  MongoClient,
+  MongoClientOptions,
+  InsertOneResult,
+  FindCursor,
+  Filter,
+  Document,
+} from "mongodb";
+
+const user = env.get("MONGO_USERNAME").required().asString();
+const pass = env.get("MONGO_PASSWORD").required().asString();
+const uri = env.get("MONGO_CLUSTER_URI").required().asString();
+const db = env.get("MONGO_DB").required().asString();
```

Comment:
`MONGO_DB` is being imported in two different locations. Both in the `api.ts` file and here.

An env variable should ideally only get imported in one area of the code.